ui/src/app/pages/analysis/runtime-panel.spec.tsx

```diff
@@ -159,6 +159,55 @@ describe('RuntimePanel', () => {
     expect(runtimeApiStub.runtime.gceConfig.machineType).toEqual('n1-standard-4');
   });
 
+  it('should create runtime with preset values instead of getRuntime values if configurationType is GeneralAnalysis', async() => {
+    const gceConfig = defaultGceConfig();
+    gceConfig.machineType = 'n1-standard-16';
+    gceConfig.diskSize = 1000;
+    const runtime = {...runtimeApiStub.runtime,
+      status: RuntimeStatus.Deleted,
+      configurationType: RuntimeConfigurationType.GeneralAnalysis,
+      gceConfig: gceConfig,
+      dataprocConfig: null
+    };
+    runtimeApiStub.runtime = runtime;
+    act(() => { runtimeStore.set({runtime: runtime, workspaceNamespace: workspaceStubs[0].namespace}); });
```

Comment:
nit: determined we didn't `act` here and dropped from other cases, probably remove here too for consistency.

Specifically, it's only needed after we render a React component.